cobra/core/Metabolite.py

```diff
@@ -167,6 +211,21 @@ def summary(self, **kwargs):
         except ImportError:
             warn('Summary methods require pandas/tabulate')
 
+    def _repr_html_(self):
```

Comment:
If its possible, could we make stuff like this a different pull request? This one seems like a pretty monolithic 'cobra becomes cameo' one, where I think it would be easier to digest if we did one for solvers, one for `___str__` methods, etc.